torch/_inductor/config.py

```diff
@@ -64,6 +64,10 @@
 # Optimize away split cat patterns (Experimental)
 split_cat_fx_passes = True
 
+# Optimize conv-bn if bn is in eval mode
+# It works for training/validation/inference
+efficient_conv_bn_eval_fx_passes = True
```

Comment:
Make this False by default

Reply:
Fixed in [03498e7](https://github.com/pytorch/pytorch/pull/108757/commits/03498e71a176dc778d2d30c87358566915081aef).